src/components/VmDetails/cards/NicsCard/index.js

```diff
@@ -123,7 +123,7 @@ class NicsCard extends React.Component {
       <BaseCard
         icon={{ type: 'pf', name: 'network' }}
         title={msg.nic()}
-        editTooltip={msg.nicCardEditTooltip({ vmName: vm.get('name') })}
+        editTooltip={msg.edit()}
```

Comment:
Same as for `disksCardEditTooltip`. I think it's better to see in the tooltip what exactly we are going to edit.

Reply:
Like I mentioned [here](https://github.com/oVirt/ovirt-web-ui/pull/1289#discussion_r499566560), that were the document's instructions,
if we want to change anything let's make a meeting and discuss that.

Reply:
Changing disksCardEditTooltip and nicCardEditTooltip text was suggested by @lwrigh (yellow sections within  [Tooltips Document](https://docs.google.com/document/d/1ouQ7b4tKH1ksmB54hnPz69rkHcynv7qhG3hDXf342XA/edit#)) and I think it's ok since it's pretty obvious that we are editing those entities for the VM. Shorter info text is better.